jupyter_book/commands/__init__.py

```diff
@@ -225,6 +225,36 @@ def toc(path, filename_split_char, skip_text, output_folder):
     _message_box(f"Table of Contents written to {output_file}")
 
 
+@main.command()
+@click.argument("path-book")
+@click.option("-a", "--all", "all_", is_flag=True, help="Remove build directory.")
+def clean(path_book, all_):
+    """Empty build directory except jupyter_cache subdirectory."""
+
+    PATH_OUTPUT = Path(path_book).absolute()
+    if not PATH_OUTPUT.is_dir():
+        _error(f"Path to book isn't a directory: {PATH_OUTPUT}")
+
+    build_path = PATH_OUTPUT.joinpath("_build")
+    if not build_path.is_dir():
+        _error(f"Your book does not have a _build directory.")
+
+    if all_:
+        # Remove .jupyter_cache
+        sh.rmtree(build_path)
+        _message_box(f"Your _build directory has been removed")
+    else:
+        # Empty _build except .jupyter_cache
+        to_remove = [
+            d for d in build_path.iterdir() if d.is_dir() and d.name != ".jupyter_cache"
```

Comment:
minor suggestion - instead of using a single letter `d`, use double-letters `dd`. That way the code is much easier to `ctrl-f` find (see how many results show up for `d` vs `dd` :-) ) (even better, use a human-readable word but the double-letter thing is a nice trick nonetheless).

Reply:
I went with the double-letter abbreviation.